crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,183 @@
+# Crosswalk for Citation File Format 1.2.0
+
+## Format information
+
+Citation File Format (CFF) is a YAML format for providing citation metdaata for software (and experimentally for datasets).
+
+- Website: <https://citation-file-format.github.io/>
+- Specifications: <https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md>
+- Schema: <https://github.com/citation-file-format/citation-file-format/blob/main/schema.json> (JSON Schema)
```

Comment:
same

Reply:
Done in 60001e2b4f55ea3b73d124b325b8a821f82eabf8.